Wire.go

```diff
@@ -73,6 +73,7 @@ import (
 	"github.com/devtron-labs/devtron/internal/util"
 	"github.com/devtron-labs/devtron/internal/util/ArgoUtil"
 	"github.com/devtron-labs/devtron/pkg/app"
+	"github.com/devtron-labs/devtron/pkg/app/status"
```

Comment:
why is status in app and not pkg/pipeline? Also, can we rename it to something more specific, status sounds too generic, hinders readability when reading the wire bindings

Reply:
have put status in app because of wiring issue which I was facing earlier, also, it was made to sound generic, like app/status of devtron app as well as helm charts(since they are also apps).